packages/ui/src/components/editor/properties/liquidcode/index.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { PopupMenuServices } from '@ir-engine/client-core/src/user/components/UserMenu/PopupMenuService'
+import { UserMenus } from '@ir-engine/client-core/src/user/UserUISystem'
+import { getComponent, hasComponent, useComponent, useEntityContext, UUIDComponent } from '@ir-engine/ecs'
+import { commitProperty, EditorComponentType, updateProperty } from '@ir-engine/editor/src/components/properties/Util'
+import { EditorControlFunctions } from '@ir-engine/editor/src/functions/EditorControlFunctions'
+import NodeEditor from '@ir-engine/editor/src/panels/properties/common/NodeEditor'
+import {
+  InteractableComponent,
+  XRUIActivationType
+} from '@ir-engine/engine/src/interaction/components/InteractableComponent'
+import { getEntityErrors } from '@ir-engine/engine/src/scene/components/ErrorComponent'
+import { LiquidCodeComponent } from '@ir-engine/engine/src/scene/components/LiquidCodeComponent'
+import { CodeSnippet01Md } from '../../../../icons'
+import InputGroup from '../../input/Group'
+import { ControlledStringInput } from '../../input/String'
+
+export const LiquidCodeReactor = () => {
```

Comment:
since this is being consumed in the client, let's move this function to client-core